### ISSUE TITLE
read_obj returns a face list of size 1 x 3 * n_faces, instead of 3 x n_faces

### ISSUE THREAD

**Issue by rubenwiersma** [rusty1s/pytorch_geometric#1571]
## 🐛 Bug

When using `read_obj` from `torch_geometric.io` to load an obj file, the `face` field of the returned `data` object is of size `1 x 3 * n_faces` instead of `3 x n_faces`. This was not yet the case in v. 1.4.3.

## To Reproduce

1. Load obj file with `torch_geometric.io.read_obj`
2. `print(data.face.size())`
3. output: `torch.Size([1, n_faces * 3])` (in this case [1, 1500])

## Expected behavior

output: `torch.Size([3, n_faces])`. This *is* correct for `read_ply`.

## Environment

* OS: Linux (Ubuntu 20.04.1)
* Python version: 3.7.4
* PyTorch version: 1.6.0
* CUDA/cuDNN version: 10.2
* GCC version: 9.3.0
* Any other relevant information: PyG version: 1.6.1

## Additional context
I'm investigating the cause and will submit a pull request when I've fixed the issue.

**Comment by rubenwiersma** [comment]
Simple fix is to also use `openmesh` for `read_obj`, other option is to fix the file reader implemented in `read_obj`. Not sure what you would prefer: less code to maintain, or fewer dependencies.

**Comment by rubenwiersma** [comment]
I traced back the bug to the following commit: #01c68f9b58c94d9efd1f6e39b9c85177aae521bb
On line 15:
```
yield ['f', [[int(i) - 1 for i in t.split("/")] for t in triangle]]
```
`t` is split, as faces can have indices for geometric vertices, texture vertices, and vertex normals. This piece of code uses every one of the indices, instead of only the geometric vertices (which are the only ones currently supported in PyTorch Geometric). It makes more sense to jsut use t.split("/")[0].

Reverting to #d42a690fba68005f5738008a04f375ffd39bbb76 would fix it.

**Comment by rusty1s** [comment]
Thanks for this issue and your valuable bug tracing. This is very helpful! I fixed the code by using `t.split("/")[0]`. For later releases, we can think of using an external library to allow for faster loading and less code maintenance.

I'm closing this issue for now, feel free to re-open in case there are any other concerns.

**Comment by simofoti** [comment]
Hi there!

I am afraid that the issue is still there. The last fix introduced a different bug in the face parsing, but as mentioned by @rubenwiersma, it can be easily fixed reverting to [this version d42a690](https://github.com/rusty1s/pytorch_geometric/blob/d42a690fba68005f5738008a04f375ffd39bbb76/torch_geometric/io/obj.py), which is the version before the attempt of parsing also indices for textures and normals (a feature which doesn't seem to be useful unless the obj_reader is further improved).

**Comment by rusty1s** [comment]
Ok, I reverted to the old version for now :)
